docs/management.md

```diff
@@ -293,42 +299,63 @@ rabbitmqctl set_user_tags full_access administrator
 * [Auth0](https://auth0.com/)
 * [Azure](https://docs.microsoft.com/en-us/azure/active-directory/fundamentals/auth-oauth2)
 * [OAuth2 Proxy](https://oauth2-proxy.github.io/oauth2-proxy/)
+* [Okta](https://www.okta.com)
 
-**Important**: from the OAuth 2.0 point of view, the RabbitMQ Management UI is a **public app** which
-means it cannot securely store credentials such as the *client_secret*. This means that RabbitMQ does not need to present a client_secret when authenticating users.
 
-It is usually possible to configure the OAuth client as a **public app** with the authorization server that you are using.
-If target authorization server only supports a **confidential app** or it requires a *client_secret*,
-then a *client_secret* **must** be configured using the `oauth_client_secret` setting.
+### Basic configuration {#basic-configuration}
 
-To redirect users to the UAA server to authenticate, use the following configuration:
+Given the following configuration of the [OAuth 2.0 plugin](./oauth2):
+
+```ini
+auth_oauth2.resource_server_id = new_resource_server_id
+auth_oauth2.issuer = https://my-oauth2-provider.com/realm/rabbitmq
+```
+
+Apply the following configuration to enable OAuth 2.0 authentication:
 
 ```ini
 management.oauth_enabled = true
 management.oauth_client_id = rabbit_user_client
-management.oauth_client_secret = rabbit_user_client
-management.oauth_provider_url = https://my-uaa-server-host:8443/uaa
 management.oauth_scopes = openid profile rabbitmq.*
 ```
 
-> **Important**: UAA supports regular expression in scopes, e.g. `rabbitmq.*`. The above configuration
-assumes that the `resource_server_id` configured in the oauth2 backend matches the value `rabbitmq`.
+- `oauth_enabled` is mandatory and it enables OAuth 2.0 authentication provided we have OAuth 2.0 plugin configured too.
+- `oauth_client_id` is mandatory and it is the OAuth Client Id associated to this RabbitMQ cluster in the OAuth Provider and it is used to request a token on behalf of the user
+- `oauth_scopes` most of the time this setting is mandatory. It is a list of space-separated strings indicating which permissions the application is requesting. Most OAuth providers will only issue tokens with the scopes requested during the user authentication. RabbitMQ sends this field along with its `oauth_client_id` during the user authentication. However, there are OAuth providers who may automatically grant scopes associated to the `oauth_client_id`. Only in this latter case, we do not need to specify the `oauth_scopes` field.
+
+If `management.oauth_scopes` is not set, RabbitMQ defaults to `openid profile`.
 
-> **Important**: Since RabbitMQ 3.10, RabbitMQ uses `authorization_code` grant type. `implicit` flow is deprecated.
+Depending on the Authorization server, you may use regular expression in scopes, e.g. `[resource_server_id].*`, or
+instead you have to explicitly ask for them, e.g.: Below `rabbitmq` value is used as the resource_server_id.
```

Comment:
I don't know what "you have to explicitly ask for them" means??

 instead of **e.g.: Below `rabbitmq` value is used as the resource_server_id.** how about: ...."instead you must request them. For example, the `rabbitmq` value is used as the resource_server_id in the following example.

not sure if "instead you must request them" is correct